network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/VoltageLevelTest.java

```diff
@@ -17,6 +17,78 @@
  */
 public class VoltageLevelTest {
 
+    @Test
+    public void testBusBreakerSetVUpdateVoltageLevel() {
+        Network network = CreateNetworksUtil.createBusBreakerNetworkWithLine();
+        LineImpl l1 = (LineImpl) network.getLine("L1");
+
+        // Update voltage using BusView
+        l1.getTerminal1().getBusView().getBus().setV(222);
+
+        // Verify the voltage update in BusBreakerView
+        assertEquals("Voltage should match in BusBreakerView after update in BusView", 222, l1.getTerminal1().getBusBreakerView().getBus().getV(), 0.0);
+
+        // Set voltage using BusBreakerView
+        l1.getTerminal1().getBusBreakerView().getBus().setV(400.0);
+
+        // Verify voltage update in BusView
+        assertEquals("Voltage should match in BusView after update in BusBreakerView", 400.0, l1.getTerminal1().getBusView().getBus().getV(), 0.0);
+    }
+
+    @Test
+    public void testNodeBreakerSetVUpdateVoltageLevel() {
+        Network network = CreateNetworksUtil.createNodeBreakerNetworkWithLine();
+        LineImpl l1 = (LineImpl) network.getLine("L1");
+
+        // Update voltage using BusBreakerView
+        l1.getTerminal1().getBusBreakerView().getBus().setV(222);
+
+        // Verify the voltage update in BusView
+        assertEquals("Voltage should match in BusBreakerView after second update in BusView", 222, l1.getTerminal1().getBusView().getBus().getV(), 0.0);
+
+        // Set voltage using BusView
+        l1.getTerminal1().getBusView().getBus().setV(400.0);
+
+        // Verify voltage update in BusBreakerView
+        assertEquals("Voltage should match in BusBreakerView after update in BusView", 400.0, l1.getTerminal1().getBusBreakerView().getBus().getV(), 0.0);
+    }
+
+    @Test
+    public void testBusBreakerSetAngleUpdateVoltageLevel() {
+        Network network = CreateNetworksUtil.createBusBreakerNetworkWithLine();
+        LineImpl l1 = (LineImpl) network.getLine("L1");
+
+        // Update angle using BusView
+        l1.getTerminal1().getBusView().getBus().setAngle(111);
```

Comment:
for better coverage, start the angle busbreaker topology test with a setAngle in the busbreakerview
and the nodebreaker topology test start with a setAngle in the busview